share_app/src/main/res/values/strings.xml

```diff
@@ -5,7 +5,9 @@
     <string name="stop">Stop</string>
     <string name="hotspot_settings_dialog">Enable portable hotspot from the settings.</string>
     <string name="hotspot_name_suffix">-odk-share</string>
+    <string name="hotspot_name_prefix_oreo">AndroidShare</string>
```

Comment:
This should probably be ODKShare?

Reply:
Ah, I see this is merged. Probably not a big deal but something to consider changing at some point!

Reply:
@lognaturel For oreo devices we can't change its hotspot configuration so when we enable/disable the hotspot, it creates a hotspot with some SSID like `AndroidShare-XXXX` where `XXXX` is some 4 digits random number.

Reply:
I see! That wasn't clear to me from the issues you linked to. You probably then want to make sure both `hotspot_name_prefix_oreo` and `hotspot_name_suffix` are in an untranslated file rather than in `strings.xml`. It might also be helpful to add a comment to make it clear that this is a value set by Android.

Reply:
Sure, will make the changes in the PR. Thanks for pointing this out.